website/docs/r/vapp.html.markdown

```diff
@@ -137,3 +142,6 @@ The following arguments are supported:
 * `org` - (Optional; *v2.0+*) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
 * `vdc` - (Optional; *v2.0+*) The name of VDC to use, optional if defined at provider level
 * `accept_all_eulas` - (Optional; *v2.0+*) Automatically accept EULA if OVA has it. Default is `true`
+* `properties` - (Optional; *v2.5+*) Key value map of vApp guest properties **Note** `ovf` attribute
```

Comment:
As I am not very enlighten with this functionality - I didn't follow this comment.

Reply:
The ugly part is that I don't find a link vCD oficial documentation about it (although this option is available in UI). Didn't want to put something like a blog link into docs https://blogs.vmware.com/vsphere/2012/06/leveraging-vapp-vm-custom-properties-in-vcloud-director.html

Reply:
Yes, this blog is a bit too old:
> one of the new features in the vCloud Director 1.5 release

:)

Reply:
The above blog has useful description how **vApp** level guest properties behave (that's the non-intuitive part):
> Custom Properties can be defined at the vApp level which will then be cascaded down to all Virtual Machines within the vApp